CONTRIBUTING.md

```diff
@@ -1,3 +1,456 @@
-Pull requests and bug reports are welcome!
+# Contributing Guidelines
 
-For larger changes please create an Issue in GitHub first to discuss your proposed changes and possible implications.
+Welcome to Function Clarity. We are excited about the prospect of you joining our community! The Function Clarity communitry abides by the CNCF [code of conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md). Here is an excerpt:
```

Comment:
Please remove the reference to the CNCF Code of Conduct.
We cannot adopt a CoC for a foundation to which we do not belong.

I've filed https://github.com/openclarity/.github/issues/3 where we can discuss, which would be part of https://github.com/openclarity/.github/issues/2#issuecomment-1379556711.